### DISCUSSION TITLE
Message limit

### DISCUSSION THREAD

**Discussion by maciej123551** [flyerhq/flutter_chat_ui#88] [Ideas & Requests]
Would be nice to limit message length with letter internationalized counter ex. 
Number of letters 15/200, and to disable entering letters when max was encountered.
like:
![image](https://user-images.githubusercontent.com/89849608/131669786-2e8bb191-caec-4a7f-a318-bfe8e6f1b1e3.png)

**Comment by larsbloch** [comment]
This could be a cool feature. We need some form of limit for the message sent to the backend.,

**Reply by demchenkoalex** [reply]
I am working on a complete package re-design, will make sure to include this feature in `v2`.
